content/v2.0/influx-templates/_index.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: Influx Templates
 description: >
-  placeholder
+  Influx Templates are prepackaged InfluxDB configurations that contain
```

Comment:
InfluxDB templates are prep....

Reply:
They are calling them "Influx templates", not "InfluxDB templates." I was treating it as a proper noun in this draft, but I've updated them all.

Reply:
in the UI? @russorat, is this something we can change? ..."templates" or "InfluxDB templates" or "InfluxData templates"?